npachpande/async/miawCustomPrechat/customPrechatFormField.html

```diff
@@ -0,0 +1,18 @@
+<template>
+    <template lwc:if={isTypeChoiceList}>
```

Comment:
ChoiceLists can be rendered using `lightning-combobox` while the rest of the fields can simply be `lightning-input` with the appropriate value for `type` attribute.